Rover/Parameters.h

```diff
@@ -404,6 +404,11 @@ class ParametersG2 {
 
     // FS options
     AP_Int32 fs_options;
+
+#if HAL_AIS_ENABLED
+    // AIS Lib
```

Comment:
Huh.  Would *never* have guessed that `AP_AIS ais;` was the AIS library!

Perhaps wrapping it in `HAL_AIS_ENABLED` helps....

```suggestion
    // Automatic Identification System - for tracking sea-going vehicles
```

make 'em useful or leave 'em out.